test/gen_source_callbacks.rb

```diff
@@ -56,18 +56,17 @@ def parse_log(input_path, yaml_path = nil)
 
   File.open(input_path, 'r') do |file|
     file.each_line.map do |line|
-      # Line format support checks
-      match = line.match(REGEXT_PRETTY)
+      match = line.match(/(?:\[(\S+)\] \(\S+\) )?(\S+): (?:{(.*)})?/)
       raise "Unsupported format for '#{line}'." unless match
-
-      head, tail = line.match(/(\S+): {(.*)/).captures
-      field_values_ts = line.match(/^\[(\S+)\]/) do |m|
-        t = Time.parse(m.to_s)
+      timestamp, head, tail = match.captures()
+      field_values_ts = nil
+      if timestamp
+        t = Time.parse(timestamp)
```

Comment:
Maybe remove the if and always do:
`t = Time.parse(timestamp || 0)` I think that should work